app/views/reviews/create.js.erb

```diff
@@ -1,3 +1,9 @@
-document.querySelector('#review-form').reset()
-document.querySelector('#review-count').innerHTML = "<%= review_count(@review.movie) %>"
-document.querySelector('#flash-id').innerHTML = "<%= j render partial: 'layouts/flash' %>"
\ No newline at end of file
+<% if !@review.errors.any? %>
```

Comment:
You can just call `@review.valid?` here. Calling `valid?` will return `true/false`, and populate `errors` collection in the background.